Marlin/src/inc/SanityCheck.h

```diff
@@ -2156,8 +2156,12 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
 /**
  * G35 Assisted Tramming
  */
-#if ENABLED(ASSISTED_TRAMMING) && !HAS_BED_PROBE
-  #error "ASSISTED_TRAMMING requires a bed probe."
+#if ENABLED(ASSISTED_TRAMMING)
+  #if !HAS_BED_PROBE
+    #error "ASSISTED_TRAMMING requires a bed probe."
+  #elif BOTH(ASSISTED_TRAMMING_MENU_ITEM, ASSISTED_TRAMMING_WIZARD)
+    #error "Please select either ASSISTED_TRAMMING_MENU_ITEM or ASSISTED_TRAMMING_WIZARD, not both."
+  #endif
```

Comment:
This is better than the undocumented current behavior.

Maybe we should instead just allow both to be enabled at the same time? They are somewhat redundant, but still different...

Reply:
We could have both, but I don't really understand why do we even have `ASSISTED_TRAMMING_MENU_ITEM` in general as that is only a `G35` gcode button, and without Pronterface, Octoprint etc. there is no use to it.

Reply:
If we have to use Pronterface, Octoprint etc. anyway for `G35`, then `ASSISTED_TRAMMING_MENU_ITEM` has no place in the printer menu in my opinion.
Approaching the problem from a different perspective, why do we have a menu item that is unusable with only the printer.

Reply:
I think you are right. If it doesn’t report information back to the screen it doesn’t provide any real value. It seems unlikely that people would want to monitor it over serial, but don’t also have a way to send G35 over serial.